src/pages/Landing/index.tsx

```diff
@@ -384,7 +386,14 @@ export default function Landing() {
                 <LearnMoreArrow />
               </LearnMoreContainer>
 
-              <DownloadWalletLink href="https://wallet.uniswap.org/">
+              <DownloadWalletLink
+                href={getDownloadAppLink()}
+                onClick={(e) => {
+                  // prevent and run command to send analytics events, but keep href for browsers to show right link
+                  e.preventDefault()
+                  openDownloadApp(undefined, downloadLinkOptions)
```

Comment:
didn't want to add analytics event but will do that now...

Reply:
pending here, we can follow on with it https://github.com/Uniswap/analytics-events/pull/47

Reply:
What is this event capturing vs the other existing one / the options?